spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -65,7 +65,7 @@ class ShellWidget(NamepaceBrowserWidget, HelpWidget, DebuggingWidget,
     sig_change_cwd = Signal(str)
 
     # For printing internal errors
-    sig_exception_occurred = Signal(str, bool)
+    sig_exception_occurred = Signal(dict)
```

Comment:
Please complete this signal using a docstring and include the interface of the `dict`.
```python
sig_exception_occurred = Signal(dict)
"""
.......
"""
```

Please also follow https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
No need to fix.